ipatests/test_xmlrpc/test_schema_plugin.py

```diff
@@ -0,0 +1,143 @@
+# Authors:
+#   Felipe Barreto <fbarreto@redhat.com>
+#
+# Copyright (C) 2012, 2017  Red Hat
+# see file 'COPYING' for use and warranty information
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
We use a standard contributors header now:

Copyright (C) 2017  FreeIPA Contributors see COPYING for license